apps/rush-lib/src/data/EnvironmentConfiguration.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+const ENV_VALUE_PREFIX: string = 'rush_';
+
+export enum EnvironmentValue {
```

Comment:
>EnvironmentValue [](start = 12, length = 16)

This is technically a key not a "value".  Maybe call the enum  "EnvironmentVariableNames"?

Reply:
Also use const enum

---
In reply to: [177928899](https://github.com/Microsoft/web-build-tools/pull/585#discussion_r177928899) [](ancestors = 177928899)

Reply:
`const enum` makes it no longer a key-value pair.

---
In reply to: [177928911](https://github.com/Microsoft/web-build-tools/pull/585#discussion_r177928911) [](ancestors = 177928911,177928899)